pkg/standalone/standalone.go

```diff
@@ -61,11 +61,11 @@ type configuration struct {
 	} `yaml:"metadata"`
 	Spec struct {
 		Tracing struct {
-			SamplingRate string `yaml:"samplingRate"`
+			SamplingRate string `yaml:"samplingRate,omitempty"`
 			Zipkin       struct {
-				EndpointAddress string `yaml:"endpointAddress"`
-			} `yaml:"zipkin"`
-		} `yaml:"tracing"`
+				EndpointAddress string `yaml:"endpointAddress,omitempty"`
+			} `yaml:"zipkin,omitempty"`
+		} `yaml:"tracing,omitempty"`
 	} `yaml:"spec"`
```

Comment:
will `omitempty` work as expected? or does dapr expect the spec property to be there for sure?

Reply:
See the test for what the config output looks like. I have validated that this does in fact work. I ran hello-world quickstart and everything works but persistence.

Reply:
the question here is if `omitempty` were added for the `Spec` field, will it still work? Wouldn't that remove the empty `spec: {}` in the file?